### ISSUE TITLE
@next: `useStoryBridge` misses errorHandler functionality

### ISSUE THREAD

**Issue by madebyfabian** [storyblok/storyblok-nuxt#77]
## Issue
Currently, it's not possible to define any sort of error handler for the `useStoryBridge` composable. <br>This leads to 
- `You are not in the edit mode.` written in the console on every page visit when I am in dev mode. This can get confusing, because it's not an error by itsself.
- I cannot create any form of manual error handling.
- I cannot mute errors.

## Proposal for Solution
Adding another parameter to the useStoryBridge function.
```vue
<script setup>
  // ...
  onMounted(() => {
    useStoryBridge(state.story.id, story => (state.story = story), {}, error => {
      // Handle error here, or just return nothing
    })
  })
</script>
```

If you agree with this, i would create a PR for it.

**Comment by madebyfabian** [comment]
Would fix #74 too

**Comment by alvarosabu** [comment]
Hi, @madebyfabian thank you for raising this and I apologize it took too long to get an answer. 

The `useStoryBridge` actually belongs to the main js SDK https://github.com/storyblok/storyblok-js/blob/main/lib/index.ts. 

I will close it here but feel free to open a ticket on the JS SDK for this if it's still relevant for you. Thanks
